config/routes.rb

```diff
@@ -3,6 +3,12 @@
   root 'restaurants#index'
 
   resources :users, only: [:new, :create]
+  resources :sessions, only:[:create]
+
+  get 'login' => 'sessions#new'
+  get 'logout'=> 'sessions#destroy'
```

Comment:
Surrounding space missing for operator `=>`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.